pkg/scheduler/core/extender.go

```diff
@@ -94,10 +97,47 @@ func NewHTTPExtender(config *schedulerapi.ExtenderConfig) (algorithm.SchedulerEx
 	}, nil
 }
 
+func (h *HTTPExtender) SupportsPreemption() bool {
+	return len(h.preemptVerb) > 0
+}
+
+// ProcessPreemption returns filtered candidate nodes and victims after running preemption logic in extender.
+func (h *HTTPExtender) ProcessPreemption(
+	pod *v1.Pod,
+	nodeToVictims map[string]*schedulerapi.Victims,
+	nodeNameToInfo map[string]*schedulercache.NodeInfo,
```

Comment:
nodeNameToInfo is unused except for the test code, can we drop it from the arguments?

Reply:
Yep, in another PR, I added TODO to remove this parameter.